Pokemon Unity/Assets/Scripts2/Pokemon/Pokemon.cs

```diff
@@ -749,6 +809,15 @@ public Abilities[] getAbilityList()
                 return this.Nature == nature;
             }
         }
+
+        /// <summary>
+        /// Returns the Nature for the SeriPokemon class to serialize Nature
+        /// </summary>
+        /// <returns></returns>
+        public Nature getNature()
```

Comment:
couldnt you just make `Pokemon.Nature` a "public get" instead of adding a new method to code bank?

Reply:
Thought I would break Rule 3: "you can make modifications to game code to resolve problems BUT cannot rewrite the core code to compensate for saving". I figured this meant not touching the { get; set; }'s since they're there for a reason (took it under "core code")

Reply:
Hahaha, I can see how that can be easily be confusing. It isn't a big deal, just figured that if the nature call has 25 references (random number) and the getNature has 1 reference. It might lead to a lot of code doing the same thing as previous code and confuse those writing new content. Again, it's not really a big deal, just thought it was interesting and curious to find your intention